pkg/controller/build/build_controller.go

```diff
@@ -155,7 +155,7 @@ func (r *reconcileBuild) Reconcile(ctx context.Context, request reconcile.Reques
 			newErrorRecoveryAction(),
 			newErrorAction(),
 		}
-	case v1.BuildStrategyRoutine:
+	case v1.BuildStrategyRoutine + v1.BuildStrategyMvnd:
```

Comment:
I suspect this should be `case v1.BuildStrategyRoutine, v1.BuildStrategyMvnd:` instead.